ios/CocoaPods/Filament.podspec

```diff
@@ -50,12 +50,12 @@ Pod::Spec.new do |spec|
   end
 
   spec.subspec "gltfio_core" do |ss|
-    ss.source_files = "include/gltfio/*.h"
+    ss.source_files = "include/gltfio/**/*.h"
```

Comment:
gltfio doesn't have nested directories for its public headers

Reply:
https://github.com/google/filament/blob/main/libs/utils/include/utils/Mutex.h#L23 that looks like include of a nested directory to me

Reply:
That's in `utils`, not `gltfio`.

Reply:
🤦‍♂️ sorry, you're right. Let me adjust that

Reply:
done